layouts/shortcodes/authors.html

```diff
@@ -1 +1 @@
-{{ trim (replaceRE "(?m)^#.+?$" "" (readFile "AUTHORS")) "\n" }}
+{{ replaceRE "([^\n]+)\n*$|\n*(.+\n)\n*" "$1$2" (replaceRE "(?m)^#.+?$" "" (readFile "AUTHORS"))  }}
```

Comment:
Could you maybe check my regex, before I merge @baltpeter?

Reply:
Oh, that is disgusting. :D But, seems ok for what it's supposed to do and if it's necessary… :/